### ISSUE TITLE
Compare Hawaii vs Hawaii + BeetlePalooza for training

### ISSUE THREAD

**Issue by samuelstevens** [samuelstevens/beetle-traits#5]
**Question**: does BeetlePalooza help models learn how to do elytra length prediction?

## Summary

1. Train models with direct coordinate prediction on Hawaii only. Use DINOv3 with a frozen encoder. Try 3 different learning rates.
2. Train models  with direct coordinate prediction on Hawaii and BeetlePalooza. Use DINOv3 with a frozen encoder. Try 3 different learning rates.
3. Evaluate all six models on the validation split of Hawaii.
4. Report quantitative metrics and qualitative examples.

## Outputs

1. A table summarizing the inputs to the model. This includes the total number of training samples, how long you trained for (in epochs), the batch size and all the learning rates.
2. Quantitative metrics. For the best learning rate  for each dataset config, based on validation loss, report
    - Training loss
    - Validation loss
    - Mean validation error in cm for width and length
    - Max validation error in cm for width and length
3. Qualitative examples. For the best learning rate for each dataset config, based on validation loss, show:
    - The 10 worst validation predictions (images)
    - 10 randomly sampled validation predictions
    - 10 samples that one config did very well on and that the other config did very poorly on

Report these in this thread using markdown. Include links to the wandb runs.

## Gotchas

1. The frozen model doesn't actually freeze the ViT. Figure out how to have models report which parameters they want optimized. Look into equinox's [tutorial on this topic](https://docs.kidger.site/equinox/examples/frozen_layer/). 
2. Use an MLP with one hidden activation vector (so the input is size $D$, then a linear layer to project to $D$, then a linear layer to project to $8$) and an GeLU activation function after the hidden activation.
3. We should not train on BeetlePalooza elytra width. Somehow we need to not calculate gradients w.r.t. BeetlePalooza elytra width loss.
